aptos-move/framework/aptos-framework/sources/governed_gas_pool.move

```diff
@@ -41,6 +54,23 @@ module aptos_framework::governed_gas_pool {
     struct GovernedGasPoolExtension has key {
         deposited_treasury_counter: u64,
         withdraw_staking_reward_events: EventHandle<WithdrawStakingRewardEvent>,
+        /// Event handle for low-balance alerts
+        low_balance_events: EventHandle<PoolLowBalance>,
+        /// Whether we've already emitted a low-balance alert for the current state
+        low_event_emitted: bool,
+    }
```

Comment:
If the GovernedGasPoolExtension is already present in the mainnet db , adding fields will break the framework upgrade. We should verify if it's exist and if yes create a new struct to allow the upgrade